test/influx-integration.spec.js

```diff
@@ -444,6 +444,34 @@ describe('@integration influxdb tests', () => {
             });
         });
 
+        describe('lag', () => {
+            it('successfully retrieves lagging points', () => {
+                // Start inserting points every 0.1s with 0.5s timestamp lag
```

Comment:
is it reliable to use 0.5s here and the same value for `-lag` or should one be larger than the other?